examples/sampling/pyg/node_classification.py

```diff
@@ -0,0 +1,184 @@
+"""
+This script demonstrates the training and testing of a GraphSAGE model for node classification on large graphs, leveraging the 
+GraphBolt framework for efficient data handling and mini-batch processing.
```

Comment:
Highlight the major target of this example, dataloading with GB and train with PyG. Highlight the difference between the counterparts of DGL and GB examples.